readme.md

```diff
@@ -660,6 +660,30 @@ As of version `5.0.0` it uses source maps to report coverage for your actual cod
 
 ## FAQ
 
+### How do I test HTTP endpoints?
+
+AVA doesn't have an official assertion library for endpoints, but a great option is [`supertest-as-promised`](https://www.npmjs.com/package/supertest-as-promised).
+Assuming you are using express, you might set up a server for each test:
+```
+let app;
+test.beforeEach(() => {
```

Comment:
> could you explain why context is needed 

Because everything executes concurrently in AVA. Assuming you are coming from mocha, you are accustomed to this order of execution:

 - `beforeEach` for  `test 1`
 - `test 1`
 - `beforeEach` for `test 2`
 - `test 2`
 - `beforeEach` for `test 3`
 - `test 3`

With AVA, your order of operations could be:

 - `beforeEach` for `test 2`
 - `beforeEach` for  `test 1`
 - `test 1`
 - `beforeEach` for `test 3`
 - `test 2`
 - `test 3`

Note it is unlikely the order is exactly as above, the point I'm making is that you can't rely on order of execution. The only ordering you are guaranteed is that `beforeEach for test X` happens before `test X`. Where it happens in relation to other tests and their respective `beforeEach`es is not guaranteed.

If you use the `--serial` flag (`ava --serial my-test.js`), then your tests are ordered similar to mocha.